neoexchange/core/templates/core/plot_spec.html

```diff
@@ -37,8 +35,7 @@ <h1>Block: <a href="{% url 'block-view' sb_id %}">{{pk}}</a></h1>
 <div class="container" id="main">
     <!-- Import the javascript and div created by Bokeh for spectra plots -->
     {% block graphic %}
-        {{ reflec_div|safe }}
-        {{ raw_div|safe }}
+        {{ div|safe }}
```

Comment:
Does it make sense to use something less generic here ? `spectro_div` ? (turns round before paddling even further out of comfort depth...)

Reply:
valid. Could be anything we want. `spectro_div` is good. The other two were only specific because we had 2 of them, but I hear it's good practice to choose recognizable variable names :)